server/src/test/java/org/opensearch/gateway/PrimaryShardBatchAllocatorTests.java

```diff
@@ -85,61 +81,60 @@ private void allocateAllUnassignedBatch(final RoutingAllocation allocation) {
         batchAllocator.allocateUnassignedBatch(shardsToBatch, allocation);
     }
 
-    public void testMakeAllocationDecisionDataFetching() {
-        final RoutingAllocation allocation = routingAllocationWithOnePrimary(noAllocationDeciders(), CLUSTER_RECOVERED, "allocId1");
-
-        List<ShardRouting> shards = new ArrayList<>();
-        allocateAllUnassignedBatch(allocation);
-        ShardRouting shard = allocation.routingTable().getIndicesRouting().get("test").shard(shardId.id()).primaryShard();
-        shards.add(shard);
-        HashMap<ShardRouting, AllocateUnassignedDecision> allDecisions = batchAllocator.makeAllocationDecision(shards, allocation, logger);
-        // verify we get decisions for all the shards
-        assertEquals(shards.size(), allDecisions.size());
-        assertEquals(shards, new ArrayList<>(allDecisions.keySet()));
-        assertEquals(AllocationDecision.AWAITING_INFO, allDecisions.get(shard).getAllocationDecision());
-    }
-
-    public void testMakeAllocationDecisionForReplicaShard() {
-        final RoutingAllocation allocation = routingAllocationWithOnePrimary(noAllocationDeciders(), CLUSTER_RECOVERED, "allocId1");
+    public void testAllocateUnassignedBatch() {
```

Comment:
Should we name this `testInitializePrimaryShards`

Reply:
Done.